src/AppBundle/Resources/views/templates/blog.html.twig

```diff
@@ -11,8 +11,8 @@
             <div class="blog-item">
                 <div class="grid">
                     <div class="grid__item desk--five-tenths lap--one-half palm--one-whole">
-                        {% if article.image|length and article.image|first.thumbnails['600x300'] is defined %}
-                                {% set image = article.image|first %}
+                        {% if article.excerpt.images|length and article.excerpt.images|first.thumbnails['600x300'] is defined %}
```

Comment:
Does the `article.image` still exist? If yes I would probably use it as a fallback here. But that's something that's not possible with the `ArticleBundle`, right?